solutions/frontend/src/templates/layout/journey-base-page.njk

```diff
@@ -1,7 +1,7 @@
 {% extends "templates/layout/base-page.njk" %}
 {% block pageContent %}
   {% block journeyPageContent %}{% endblock %}
-  {% if globals.exitJourneyUrl %}
+  {% if globals.exitJourneyUrl and not dontShowExitJourneyLink %}
```

Comment:
not a blocker, but does it make sense to invert this? e.g. change it to a `showExitJourneyLink` parameter, if the majority of the pages don't have it?